setup.py

```diff
@@ -9,6 +9,6 @@
     package_dir={'': 'library'},
     py_modules=["manageiq_provider", "manageiq_policy_assignment",
                 "manageiq_custom_attributes", "manageiq_user",
-                "manageiq_tag_assignment"],
+                "manageiq_tag_assignment", "manageiq_utils"],
```

Comment:
Tests fail on `import manageiq as manageiq_utils`.  Not sure why, but perhaps dropping the `_utils` here will help?